tests/test_file_transfer_service.py

```diff
@@ -35,6 +35,7 @@
 )
 
 
+@pytest.mark.skip(reason="https://github.com/ansys/pyfluent/issues/4649")
```

Comment:
@hpohekar I know you're looking into this test failure, but I've disabled this for now as the latest nightly image is required to verify an issue in dev docs CI.